backend/routes/users.js

```diff
@@ -74,8 +11,13 @@ const saltRounds = 10;
 env.config();
 
 router.post("/signup", async (req, res) => {
-    const { FirstName, MiddleName, LastName, Username, Password, ConfirmPassword, Role } = req.body;
+    console.log(req.body)
```

Comment:
Let's remove the console.log. We shouldn't log things in production for fear of cluttering and exposing private information to a user's console.